src/models/WorkerPoolFilter.ts

```diff
@@ -0,0 +1,13 @@
+type any_ = { any_?: string[] }
+type operator_ = 'and_' | 'or_'
```

Comment:
I'm surprised we don't have these defined more globally?

Reply:
@pleek91, I know you've done a lot with filter types?

Reply:
We have them defined in Union Filter, but they are part of that filter, so exporting them to totally separate filter might be odd. I can extract them in separate PR, they are very useful standalone

Reply:
I believe these exist in the `UnionFilters` file but they are not exported. With the context of a branch I have that redoes all the filter types, I think adding these here self contained is preferred rather than trying to export something from union filters (which is going to be deleted in the near future hopefully).